lib/license_finder/license/definitions.rb

```diff
@@ -46,6 +46,7 @@ def apache1_1
         License.new(
           short_name: 'Apache1_1',
           pretty_name: 'Apache 1.1',
+          spdx_id: 'Apache-1.0',
```

Comment:
Is this correct? This is the license for Apache 1.1. I would expect the spdx id to be  `Apache-1.1` (from the [spdx license list](https://spdx.org/licenses/))

Reply:
totally right, I just fixed it. Thank you for reviewing !

Reply:
i doubled check the other ones, should be okay.